### ISSUE TITLE
Runtime error when running simple model

### ISSUE THREAD

**Issue by pierucci** [pierucci/heemod#99]
I get a runtime error when I run this:

```r
par1 <- define_parameters(
  a = 1,
  b = 1 / (markov_cycle + a)
)

mat1 <- define_matrix(
  1-b, b,
  0, 1
)
mod1 <- define_model(
  transition_matrix = mat1,
  define_state(var = a),
  define_state(var = a * markov_cycle)
)

res1 <- run_models(
  mod1,
  cycles = 5,
  init = 1:0,
  method = "end"
)
```

**Comment by MattWiener** [comment]
I do to - it says "object b not found".  But it looks to me as though you left the parameters (par1) out of your  run_models statement.   If I add that, and tell it that "var" is my cost or effect, things run OK:

'res1 <- run_models(
     mod1,
     cycles = 5,
init = 1:0,     method = "end", parameters = par1, cost = var, effect = var
)`

Then it seems perfectly happy, and the eval_model_newdata statement I saw elsewhere also works.

Regards,
Matt

**Comment by pierucci** [comment]
The `Error: object 'b' not found` is what I expected, but what I got was a complete runtime error that shut down R and RStudio.

Turns out it was a problem with RStudio, solved in the preview version: hadly/dplyr#2064
